DEVELOPER_GUIDE.md

```diff
@@ -42,7 +42,7 @@ Currently we just put RCF jar in lib as dependency. Plan to publish to Maven and
 2. `./gradlew :run` launches a single node cluster with anomaly-detection (and job-scheduler) plugin installed
 3. `./gradlew :integTest` launches a single node cluster with anomaly-detection (and job-scheduler) plugin installed and runs all integration tests except security
 4. ` ./gradlew :integTest --tests="**.test execute foo"` runs a single integration test class or method
-5. `./gradlew integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster" -Dhttps=true -Duser=admin -Dpassword=admin` launches integration tests against a local cluster and run tests with security
+5. `./gradlew integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="docker-cluster" -Dhttps=true -Duser=admin -Dpassword=myStrongPassword123!` launches integration tests against a local cluster and run tests with security
```

Comment:
`myStrongPassword123!` should not be considered as default password.

`admin` should be replaced with a placeholder `<admin-password>`

Reply:
Updated with placeholder `<admin-password>` in the latest revision